test/export/test_export.py

```diff
@@ -6894,17 +6884,6 @@ def test(ep):
             # running decompositions again should work for all IRs
             ep = export(M(), inp, preserve_module_call_signature=("n",))
             test(ep.run_decompositions({}))
-            if is_training_ir_test(self._testMethodName):
```

Comment:
Do these tests run in OSS mode? Are we expecting divergence when testing through fbcode?

Reply:
In fbcode, we also run the tests with training IR for our own tests: https://www.internalfb.com/diff/D65362160